packages/table/src/table.tsx

```diff
@@ -432,13 +449,23 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
     private columnHeaderCellRenderer = (columnIndex: number) => {
         const props = this.getColumnProps(columnIndex);
+        let loading: boolean;
+        switch (this.hasLoadingOption(props.loadingOptions, ColumnLoadingOption.HEADER)) {
```

Comment:
actually wait what's wrong with assigning `loading = this.hasLoadingOption(...)` directly?